moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -704,59 +756,149 @@
              </layout>
             </item>
             <item>
-             <widget class="QCheckBox" name="allow_replanning">
+             <widget class="QCheckBox" name="use_cartesian_path">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="minimumSize">
+               <size>
+                <width>30</width>
+                <height>0</height>
+               </size>
+              </property>
+              <property name="toolTip">
+               <string>Check this to generate a linear path in Cartesian (3D) space</string>
+              </property>
               <property name="text">
-               <string>Allow Replanning</string>
+               <string>Use Cartesian Path</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
-             <widget class="QCheckBox" name="allow_looking">
+             <widget class="QCheckBox" name="collision_aware_ik">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="minimumSize">
+               <size>
+                <width>30</width>
+                <height>0</height>
+               </size>
+              </property>
+              <property name="toolTip">
+               <string>If checked, a new IK solution is calculated whenever the robot state is in collision. This setting only affects the state set via the interactive marker. Planning is always aware of collisions.</string>
+              </property>
               <property name="text">
-               <string>Allow Sensor Positioning</string>
+               <string>Reset IK on collision</string>
```

Comment:
Please keep the name as-is. "Reset IK on collision" is technically also wrong, because what exactly happens depends on the IK solver.

Reply:
Ok. I had changed it because "Collision-aware" made me think "Why would I ignore collisions? Is this dangerous?" when I first saw it, but if it comes up elsewhere then I'm fine with it.